lib.php

```diff
@@ -24,6 +24,8 @@
 
 defined('MOODLE_INTERNAL') || die();
 
+include('deprecatedlib.php');
```

Comment:
Do we need this?

Reply:
Just to get rid of deprecation messages ;-)